webdriver/tests/classic/perform_actions/wheel.py

```diff
@@ -55,15 +56,12 @@ def test_scroll_iframe(session, test_actions_scroll_page, wheel_chain):
 
     wheel_chain.scroll(0, 0, 5, 10, origin=target).perform()
 
-    events = get_events(session)
+    # Chrome requires some time (~10-20ms) to process the event from the iframe, so we wait for it.
```

Comment:
FYI we might need this in Firefox as well once we dispatch events from the parent process which will add extra time for processing. I'll keep this in mind and update once that is the case.

Reply:
ack